homeassistant/components/habitica/sensor.py

```diff
@@ -1,28 +1,56 @@
 """Support for Habitica sensors."""
+from collections import namedtuple
 from datetime import timedelta
 import logging
 
+from aiohttp import ClientResponseError
+
 from homeassistant.components import habitica
```

Comment:
Don't import the whole module. Use relative imports for intra package imports and just import the used names.